test/wave-packerSpec.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable
+ camelcase,
+ new-cap
+ */
+
+/* global
+ describe,
+ expect,
+ it,
+ spyOn,
+ window,
+ FormData
+ */
+
+const WavePacker = require('../wave-packer');
+
+describe('Warning tests', function() {
+  it('Should warn when the sample rates are invalid', function() {
+    spyOn(console, 'warn');
+    var warning = '48000 or 44100 are the only supported recordingSampleRates';
+    var wavePacker = new WavePacker();
+    wavePacker.init(0, 0, 0);
+    expect(console.warn).toHaveBeenCalledWith(warning);
+    expect(wavePacker.recordingSampleRate).toBeDefined();
```

Comment:
Don't just expect it to be defined. Expect it to be the actual value that's used.